gotpm/cmd/seal.go

```diff
@@ -132,5 +144,6 @@ func init() {
 	// PCRs and hash algorithm only used for sealing
 	addPCRsFlag(sealCmd)
 	addHashAlgoFlag(sealCmd)
+	addPCRsFlag(unsealCmd)
```

Comment:
We also need to add the HashAlgoFlag here for unseal, as we want the ability to select a specific hash bank.

Reply:
No longer needed, as we will use the CertifyHashAlgTpm